config/jobs/kubernetes/sig-autoscaling/sig-autoscaling-config.yaml

```diff
@@ -293,7 +293,7 @@ periodics:
       # Enable HPAContainerMetrics. Required for container metrics tests.
       - --env=KUBE_FEATURE_GATES=HPAContainerMetrics=true
       - --extract=ci/latest
-      - --timeout=100m
+      - --timeout=300m
```

Comment:
240m Should be enough for now.

Reply:
also there is timeout in line 286. Please set its value as this timeout + 20m

Reply:
Alright, thank you!